### PR TITLE
Acolytes

### PR DESCRIPTION
**Pull request by Geevies**
A gamemode which features Cult and Borers.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Chada-1** [comment]
_[Goa'uld Music Plays in the background]_

File: code/game/gamemodes/mixed/acolytes.dm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/datum/game_mode/acolytes
+	name = "Acolytes (Borer+Cult)"
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/mixed/acolytes.dm]:
```suggestion
	name = "Acolytes (borer+cult)"
```

File: code/game/gamemodes/mixed/acolytes.dm [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+	round_description = "A cult and a borer infestation is on-station!"
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/mixed/acolytes.dm]:
```suggestion
	round_description = "A cult and a borer infestation are on-station!"
```

**Pull request merged**